ports/esp32s2/common-hal/wifi/Radio.c

```diff
@@ -194,6 +198,20 @@ mp_obj_t common_hal_wifi_radio_get_ap_info(wifi_radio_obj_t *self) {
     if (esp_wifi_sta_get_ap_info(&self->ap_info.record) != ESP_OK){
         return mp_const_none;
     } else {
+        // The struct member appears to be <null> (not NULL!), I don't know how to properly test for it.
+        // When the ESP-IDF starts working fine (when their bugfix is available), this "if" wouldn't trigger.
+        // Note: It is possible that Wi-Fi APs don't have a CC set, then even after this workaround
+        //       the element would remain empty.
+        if (strlen(self->ap_info.record.country.cc) == 0) {
```

Comment:
This test is fine. I don't think you need the comment above. Instead you can put the comment below there and remove the log lines.

Reply:
Ok, will do